test/unit/specs/fields/fieldCleave.spec.js

```diff
@@ -41,9 +42,9 @@ describe("fieldCleave.vue", function() {
 			expect(input).to.be.defined;
 			expect(input.type).to.be.equal("text");
 			expect(input.classList.contains("form-control")).to.be.true;
-			expect(input.placeholder).to.be.equal(schema.placeholder);	
-			expect(input.readOnly).to.be.false;	
-			expect(input.disabled).to.be.false;	
+			// expect(input.placeholder).to.be.equal(schema.placeholder);
+			// expect(input.readOnly).to.be.false;
```

Comment:
You are mindful. But if we covered with your new code we can delete them. Because after 3 months, we won't know it is a skipped tests (what need to fix) or unneccessary :)